src/components/__tests__/RouteItemDisplay.cy.js

```diff
@@ -142,20 +156,124 @@ function coreTests() {
       .should('be.visible')
       .invoke('width')
       .should('be.eq', 18);
+    // avatar transport
+    cy.dataCy('avatar-transport').should('be.visible');
```

Comment:
Repeatable string or number should be const.

Reply:
Extracted to a variable.